strapi/strapi
# Add Audit Logs feature

### What does it do?

Add the Audit Logs feature to Strapi, PR authors: @madhurisandbhor @remidej @markkaylor @Feranchz 

### How to test it?

* Be sure to have a valid EE license active
* Start your Strapi application
* Go to `Settings -> Audit Logs`, you will see a list with a log of events, you should see also the details of each event in a specific modal if you click the details icon at the right
* You can create events doing general actions such as create/edit/delete content-types, users, media, roles and others...
